client/src/components/DemoUserButton/DemoUserButton.tsx

```diff
@@ -0,0 +1,44 @@
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { CircularProgress } from '@material-ui/core';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import login from '../../helpers/APICalls/login';
+import { demoUser } from '../../mocks/mockUser';
+import { useState } from 'react';
+
+const DemoUserButton = (): JSX.Element => {
+  const classes = useStyles();
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
```

Comment:
We don't seem to change this state ever. It might also be better to receive this from `FormikContext` ([ref](https://formik.org/docs/api/useFormikContext)).

Reply:
Switched to FormikContext